src/Chainweb/Transaction.hs

```diff
@@ -32,7 +32,7 @@ import Control.Lens
 import qualified Data.Aeson as Aeson
 import Data.ByteString.Char8 (ByteString)
 import qualified Data.ByteString.Char8 as B
-import qualified Data.ByteString.Short as SB
+import qualified Data.ByteString.Short as SBS
```

Comment:
This change is completely unrelated to this PR, right?

Reply:
Thx, fixed!